src/block/columns/edit.js

```diff
@@ -62,15 +62,20 @@ const Edit = props => {
 	const rowClass = getRowClasses( props.attributes )
 	const separatorClass = getSeparatorClasses( props.attributes )
 	const blockAlignmentClass = getAlignmentClasses( props.attributes )
-	const { hasInnerBlocks } = useBlockContext()
+	const { hasInnerBlocks, numInnerBlocks } = useBlockContext()
 	const [ columnProviderValue, columnTooltipClass ] = ColumnInnerBlocks.useContext()
 
+	const userAgent = navigator?.userAgent
+
 	const blockClassNames = classnames( [
 		className,
 		'stk-block-columns',
 		rowClass,
 		separatorClass,
 		columnTooltipClass,
+		{
+			'stk-block-columns--has-single-block-polyfill': numInnerBlocks === 1 && userAgent?.indexOf( 'Firefox' ) !== -1,
+		},
```

Comment:
Can you please add a new filter for the `blockClassNames` similar to:

```js
const blockClassNames = classnames( applyFilters( '...', [
```

Then add do an `addFilter` on this at the bottom of this `src/block/columns/edit.js` file